src/components/Forecast/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import * as types from 'actions/types';
+
+import Spinner from 'components/Spinner';
+import ForecastDetails from 'components/ForecastDetails';
+
+import { getForecastData, getIsFetchingStatus, getLocation } from 'reducers';
+
+class Forecast extends React.Component {
+  componentDidMount() {
+    const { fetchForecast } = this.props;
+    fetchForecast();
+  }
+
+  render() {
+    const { isFetching, location, forecastData } = this.props;
+
+    return (
+      <div>
+        { isFetching
+          ? <Spinner />
+          : <ForecastDetails location={location} forecastData={ forecastData } />
+        }
+      </div>
+    )
+  }
+};
+
+Forecast.propTypes = {
+  isFetching: PropTypes.bool,
+  location: PropTypes.string.isRequired,
+  fetchForecast: PropTypes.func.isRequired,
+  forecastData: PropTypes.array.isRequired,
+};
+
+const mapStateToProps = (state) => {
+  return {
```

Comment:
Why not `(state) => ({})`?

Reply:
I was going through the files and updating those, must have missed it here. Fix incoming.